### PR TITLE
Add Thanos Project ideas

### PR DESCRIPTION
**Pull request by kakkoyun**
cc @bwplotka @squat @brancz

### CODE REVIEW

**Comment by bwplotka** [comment]
Lgtm (:

File: communitybridge/2020/q3-q4/project_ideas.md [hunk 1]
```diff
@@ -101,4 +101,26 @@ Project maintainers and mentors, please submit the ideas below (under the Propos
 -	Description: Improve Job API to support specific types of job
 -	Recommended Skills: Volcano, Job Management, Golang
 -	Mentor(s): Zhonghu Xu(@hzxuzhonghu), Klaus Ma(@k82cn) 
--	Upstream Issue (URL): https://github.com/volcano-sh/volcano/issues/1034
\ No newline at end of file
+-	Upstream Issue (URL): https://github.com/volcano-sh/volcano/issues/1034
+
+#### Thanos
+##### Receive Controller (tentative)
```
Comment (brancz) [inline communitybridge/2020/q3-q4/project_ideas.md:107]: What exactly is the project here and how does it differ from 2?

Comment (kakkoyun) [reply communitybridge/2020/q3-q4/project_ideas.md:107]: We are going to merge two projects and at the end will have one for the receiver component. I just created this draft to initialize to process. We're gonna finalize it today.

Comment (brancz) [reply communitybridge/2020/q3-q4/project_ideas.md:107]: nice that sounds good

Comment (squat) [reply communitybridge/2020/q3-q4/project_ideas.md:107]: I spoke with @kakkoyun, this was a misunderstanding I think. There was no project for TRC, but rather one for Thanos Receive.

File: communitybridge/2020/q3-q4/project_ideas.md [hunk 2]
```diff
@@ -101,4 +101,26 @@ Project maintainers and mentors, please submit the ideas below (under the Propos
...
+-	Recommended Skills: Thanos, Kubernetes, Golang
+-	Mentor(s): Bartek Plotka(@bwplotka), Kemal Akkoyun(@kakkoyun), Lucas Servén Marín(@squat), Frederic Branczyk(@brancz)
+-	Upstream Issue (URL): https://github.com/thanos-io/thanos/issues/XXX
+
+##### Receive Hashring Update improvements (tentative)
+
+-	Description: Receive Hashring Update improvements (tentative)
+-	Recommended Skills: Thanos, Timeseries database, Golang
+-	Mentor(s): Bartek Plotka(@bwplotka), Kemal Akkoyun(@kakkoyun), Lucas Servén Marín(@squat), Frederic Branczyk(@brancz)
+-	Upstream Issue (URL): https://github.com/thanos-io/thanos/issues/XXX
```
Comment (squat) [inline communitybridge/2020/q3-q4/project_ideas.md:119]: Issue: https://github.com/thanos-io/thanos/issues/3141

File: communitybridge/2020/q3-q4/project_ideas.md [hunk 3]
```diff
@@ -101,4 +101,26 @@ Project maintainers and mentors, please submit the ideas below (under the Propos
...
+
+##### UI Improvements follow-up work (tentative)
```
Comment (squat) [inline communitybridge/2020/q3-q4/project_ideas.md:121]: Issue: https://github.com/thanos-io/thanos/issues/3142

**Comment by idvoretskyi** [comment]
@kakkoyun can you please rebase?

**Comment by kakkoyun** [comment]
@idvoretskyi done 💜

**Review by idvoretskyi — APPROVED** [review]

**Pull request merged**